docs/trouble-shooting.md

```diff
@@ -0,0 +1,178 @@
+The suffix after `rad:git` will correspond to the namespace of the project — see
+[here](monorepo) for more details.
+
+Your monorepo will live under `$XDG_DATA_HOME/radicle/git`. Running the
+following command you should see something like:
+
+```
+$ tree $XDG_DATA_HOME/radicle/git/refs
+
+/home/user/.local/share/radicle/git/refs/
+├── heads
+├── namespaces
+│   ├── hwd1yre8ibmso5webrog7uqwcrys4fmdijcyjfoso73utnz7y41dbtqudxe
+│   │   └── refs
+│   │       └── rad
+│   │           ├── id
+│   │           ├── self
+│   │           └── signed_refs
+│   └── hwd1yredpofbnmb6d8ea4ofqu31fzziho8xoejjcd15p553fyr4msmg4k8w
+│       └── refs
+│           ├── heads
+│           │   └── main
+│           └── rad
+│               ├── id
+│               ├── ids
+│               │   └── hwd1yre8ibmso5webrog7uqwcrys4fmdijcyjfoso73utnz7y41dbtqudxe
+│               ├── self
+│               └── signed_refs
+└── tags
+```
+
+The suffix we pointed out earlier should correspond to one of these entries. In
+this error case the `refs/heads/main` will be missing.
+
+If you just **replicated** the project, then delete this namespace entry and try
+replicate again.
+
+If you just **created** the project, then navigate to your working copy and do
+the following:
+
+```
+$ git checkout -b <default branch>
+```
+
+Add a `README.md` and commit this to the `<default branch>`, then:
+
+```
+$ git push rad <default branch>
+```
+
+### Stats
+#### Reason
+
+The [browser](radicle-surf) for the project could not fetch the statistics for your project.
+This could be down to an error in git.
+
+#### Action
+
+Reach out to us on one of our channels, mention the `Stats` code, and provide
+as much detail as possible.
+
+### SignedRefs
+#### Reason
+
+`signed-refs` are the git references that peers advertise which they have also
+signed with their secret key. This means that when your peer receives
+`signed-refs` it attempts to verify the signature using the advertised [Device
+ID](device-id). If this verification fails, then this project may be left in a
+broken state.
+
+#### Action
+
+Some of these cases may not be errors and the `signed-refs` did not actually
+come from that Device ID. If you think this is indeed an error, then we can
+attempt to inspect the `signed-refs`.
+
+To get the JSON data for the `signed-refs` we must run the following commands,
+filling in the details for your particular project and remote.
+
+```
+$ cd $XDG_DATA_HOME/radicle/git
+
+$ git ls-tree refs/namespaces/<id>/refs[/remotes/<peer>]/rad/signed_refs
+100644 blob e3f6e6bd25955802060698f5b5449c874969aa71    refs
+
+$ git cat-file -p e3f6e6bd25955802060698f5b5449c874969aa71
+{"refs":{"heads":{},"remotes":{}},"signature":"hyydp6mkwkrx1cuz8i4pg3wfojujxaa3s81higa846msaroegcdspp3qe358wmpa3f85s6zmyha5zfgtqnkszwcc6a3k7pfd35fnceoec"}
+```
+
+If you report this JSON payload and the offending Device ID, we can take a look
+and see if we notice anything out of the ordinary.
+
+## UnexpectedBackendExit
```

Comment:
I’ve renamed “proxy” to “backend”. I think the latter is much easier to understand for users. I’m also thinking about renaming this error to `BackendTerminated` which would make it more aligned with the usual terminology. Any thoughts

Of course this needs to be followed up with a change to the app.